### ISSUE TITLE
absence of whitespaces fires a fake error E034

### ISSUE THREAD

**Issue by matiasbarone** [twbs/bootlint#181]
In the example from https://github.com/twbs/bootlint/wiki/E034 you can find this:

``` html
<div class="alert alert-warning alert-dismissible" role="alert">
  <button type="button" class="close" data-dismiss="alert"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
  Better check yourself, you're not looking too good.
</div>
```

But if you use the same **without whitespaces**, it tells you that it's an **E034** Error

``` html
<div class="alert alert-warning alert-dismissible" role="alert"><button type="button" class="close" data-dismiss="alert"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>Better check yourself, you're not looking too good.</div>
```
